man/TrackHubGenome-class.Rd

```diff
@@ -0,0 +1,112 @@
+\name{QuickloadGenome-class}
```

Comment:
Noticed that the `name` is wrong (copy and pasted from the Quickload stuff, I guess)

Reply:
I'll fix this, Yes I could not resist copying from Quickload as a lot of the content was similar :p
and from now onwards I'll be more vigilant while copying stuff to avoid these silly mistakes.